src/js/index.js

```diff
@@ -85,7 +104,12 @@ function cascade(force = false) {
 	}
 }
 
-window.addEventListener('load', () => {
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
+1 on using `DOMContentLoaded` instead of `load` 

Reference: https://stackoverflow.com/questions/2414750/difference-between-domcontentloaded-and-load-events